joomla/joomla-cms
# [4.0] Namespace com_plugins

Pull Request for Issue # .

### Summary of Changes
This simple PR convert com_plugins component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_plugins.zip )

2. Access to Extensions -> Plugins, try to update, enable, disable , checkin plugins, make sure it is working as expected
